libraries/aiModels/gemini/actions/response/common/eventStructure.md.ejs

```diff
@@ -43,6 +43,10 @@ Then, there are a couple additional properties whose presence and shape depend o
 | `parent` | `object` | A pointer to a relevant parent event.            |
 | `detail` | `object` | An object containing the application data for the event. |
 
+### Mentioning Users
+
+In message content, you can mention users using the `@handle` format (e.g., `@john_doe`). The system will automatically convert these to the correct platform-specific format when sending messages. User handles are derived from their display names/usernames in the chat space.
```

Comment:
Not exactly correct?

We should inform Bott that we know limited information about these users, and that it should only "mention" when it wants to get attention; otherwise it should try to use their display name

we may need an additional `knownUsers.md.ejs` document added to the process.